src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,51 @@
+package duke.command;
+
+import duke.exception.InvalidInputException;
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+import javax.sound.midi.SysexMessage;
+
+public class FindCommand extends Command {
+
+    private String keyword;
+    private final String tab = "      ";
+    private final String line = "------------------------------------------------------------";
+
+    public FindCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    /**
+     * Executes the specified command.
+     *
+     * @param tasks The TaskList which we are modifying.
+     * @param ui The Ui we will use for user interaction.
+     * @param storage The Storage we will use for storing save data.
+     * @throws InvalidInputException When the input is deemed invalid.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws InvalidInputException {
```

Comment:
Perhaps you can make this part more user friendly by allowing users to search for the task reagardless of capitalization. 

So in that case if i search "apple", the program should still return "Apple Eating Contest" or "Work at Apple"